src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerGetMetadataIntTest.java

```diff
@@ -114,4 +114,41 @@ void getAudioMetadataHearingNotFound() throws Exception {
         mockMvc.perform(requestBuilder).andExpect(status().isNotFound());
     }
 
+    @Test
+    void getAudioMetadataGetShouldNotReturnHiddenMediaChannel1() throws Exception {
+        var mediaChannel1 = dartsDatabase.createHiddenMediaEntity("testCourthouse", "testCourtroom", MEDIA_START_TIME, MEDIA_END_TIME, 1);
+        var mediaChannel2 = dartsDatabase.createMediaEntity("testCourthouse", "testCourtroom", MEDIA_START_TIME, MEDIA_END_TIME, 2);
+        var mediaChannel3 = dartsDatabase.createMediaEntity("testCourthouse", "testCourtroom", MEDIA_START_TIME, MEDIA_END_TIME, 3);
+        var mediaChannel4 = dartsDatabase.createMediaEntity("testCourthouse", "testCourtroom", MEDIA_START_TIME, MEDIA_END_TIME, 4);
+        dartsDatabase.getExternalObjectDirectoryStub().createExternalObjectDirectory(mediaChannel1,
+                                                                                     ObjectRecordStatusEnum.STORED,
+                                                                                     ExternalLocationTypeEnum.UNSTRUCTURED,
+                                                                                     UUID.randomUUID());
+
+        var hearingEntity = dartsDatabase.givenTheDatabaseContainsCourtCaseWithHearingAndCourthouseWithRoom(
+            "999",
+            "test",
+            "test",
+            LocalDateTime.now()
+        );
+        hearingEntity.addMedia(mediaChannel1);
+        hearingEntity.addMedia(mediaChannel2);
+        hearingEntity.addMedia(mediaChannel3);
+        hearingEntity.addMedia(mediaChannel4);
+        dartsDatabase.save(hearingEntity);
+
+        UserAccountEntity testUser = dartsDatabase.getUserAccountStub()
+            .createAuthorisedIntegrationTestUser(hearingEntity.getCourtroom().getCourthouse());
+        when(mockUserIdentity.getUserAccount()).thenReturn(testUser);
+
+        var requestBuilder = get(ENDPOINT_URL, hearingEntity.getId());
+
+        MvcResult mvcResult = mockMvc.perform(requestBuilder)
+            .andExpect(status().isOk())
+            .andReturn();
+
+        String actualJson = mvcResult.getResponse().getContentAsString();
```

Comment:
Can we add a media channel that wont be filtered and adjust the assertion according. Would prove that just hidden media are filtered.

Reply:
oh I see we've merged. No worries